java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -73,6 +73,13 @@ public static Scalar fromNull(DType type) {
       return new Scalar(type, makeTimestampTimeScalar(type.nativeId, 0, false));
     case STRING:
       return new Scalar(type, makeStringScalar(null, false));
+    case DURATION_DAYS:
+      return new Scalar(type, makeDurationDaysScalar(0, false));
+    case DURATION_MICROSECONDS:
+    case DURATION_MILLISECONDS:
+    case DURATION_NANOSECONDS:
+    case DURATION_SECONDS:
+      return new Scalar(type, makeDurationScalar(type.nativeId, 0, false));
```

Comment:
Nit: The `makeDurationScalar` name is too generic.  Why wouldn't `DURATION_DAYS` call this as well?  The name implies it should.